src/Compiler/CompilerPasses.cpp

```diff
@@ -214,7 +214,7 @@ void addONNXToKrnlPasses(mlir::PassManager &pm, int optLevel, bool enableCSE,
 
 void addKrnlToAffinePasses(mlir::PassManager &pm) {
   pm.addNestedPass<func::FuncOp>(
-      onnx_mlir::krnl::createConvertKrnlToAffinePass(enableParallel));
```

Comment:
You lost some changes from dev main. Please add them back.

Reply:
Fixed.